src/lib.rs

```diff
@@ -1,19 +1,17 @@
+use act::proclamation::Proclaim;
```

Comment:
Organize this so that things that can be in their own file are. It should mostly be just exposing thing and not declaring them